packages/documentation/copy/en/reference/Decorators.md

```diff
@@ -14,7 +14,7 @@ translatable: true
 
 With the introduction of Classes in TypeScript and ES6, there now exist certain scenarios that require additional features to support annotating or modifying classes and class members.
 Decorators provide a way to add both annotations and a meta-programming syntax for class declarations and members.
-Decorators are a [stage 2 proposal](https://github.com/tc39/proposal-decorators) for JavaScript and are available as an experimental feature of TypeScript.
+Decorators are a [stage 3 proposal](https://github.com/tc39/proposal-decorators) for JavaScript and are available as an experimental feature of TypeScript.
```

Comment:
This change kinda implies that TS supports the stage 3 version of decorators, which isn't strictly true because there's going to be differences between the current support and the official support
 

```suggestion
Decorators are a [stage 3 proposal](https://github.com/tc39/proposal-decorators) for JavaScript, and are currently available as an experimental feature of TypeScript using an older version of the specification.
```